function oi() {
    // return Promise.resolve( 'Ana' );
    return Promise.reject( new Error( 'NÃ£o pude obter o nome.' ) );
}

oi()
    .then( nome => {
        console.log( 'Oi', nome );
    } )
    .catch( erro => {
        console.error( 'ERRO: ' + erro.message );
    } );
console.log( 'Teste' );